src/processor/intents/creeps/_die.js

```diff
@@ -12,6 +12,26 @@ module.exports = function(object, roomObjects, bulk, stats, dropRate) {
     bulk.remove(object._id);
     delete roomObjects[object._id];
 
+    let tombstone = {
+        type: 'tombstone',
+        room: object.room,
+        x: object.x,
+        y: object.y,
+        user: object.user,
+        deathTime: gameTime,
+        decayTime: gameTime + (object.body.length * C.TOMBSTONE_DECAY_PER_PART),
+        creepId: object._id,
+        creepName: object.name,
+        creepBody: object.body,
+        creepAgeTime: object.ageTime,
+        creepBody: object.body,
+        creepHits: object.hits,
```

Comment:
There is no need to store creep's hits, it seems to always be 0 since the creep is dead, right? Also, `body[].hits` is all 0 too, so we can store body in plain format `['move','move','work',...]` (but expose to user in normal `Creep.body` format).

Reply:
is hits 0 when a creep dies by TTL?

Reply:
Thinking about it, people can use TTL to determine cause of death, TTL would be 0 on age death, suicides are user triggered, and death from damage would leave a positive TTL